corehq/apps/style/static/style/less/bootstrap3/forms.less

```diff
@@ -94,6 +94,11 @@ legend .subtext {
   }
 }
 
+// Static width for select2 widgets, which otherwise grow too large on case management pages
+.form-control.select2-container {
```

Comment:
@benrudolph it's not the limit. it's *setting* it to that across all reports. I'd rather not have had this merged in, i was in favor of a class for app manager select2s